mlir/tools/mlir-miopen-lib/mlir-miopen-lib.h

```diff
@@ -0,0 +1,85 @@
+//===-------------- mlir-miopen-lib.h - C API ---------------------*- C -*-===//
```

Comment:
@atamazov @sjw36 Mind take a second look at the exported header? I have given everything consistent naming after review MLIR coding conventions.